src/core/analyzer/type_inferrer.py

```
@@ -464,6 +453,44 @@ def _compute_annotation_bonus(annotation_coverage: float) -> float:
     return annotation_coverage**0.8
 
 
+def _parse_mypy_statistics(output: str) -> dict[str, dict[str, int]] | None:
+    """
+    mypyの統計情報を解析します。
+
+    Args:
+        output: mypyの標準出力
+
+    Returns:
+        パースされた統計情報の辞書、またはNone（統計情報が見つからない場合）
```

Ruff (0.14.0)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)